### PR TITLE
Worked on 3 out 4 fixes of the send reminder feature.

### PR DESCRIPTION
**Pull request by Jonyango**

### CODE REVIEW

**Comment by isaacadariku** [comment]
@ufe-pr Please what do you think, its causing the test running async

File: lib/ui/views/business/settings/settings_page/settings_page_viewmodel.dart [hunk 1]
```diff
@@ -8,7 +8,7 @@ import 'package:mycustomers/core/services/bussiness_setting_service.dart';
 import 'package:mycustomers/ui/shared/themes.dart' as themes;
 
 class SettingsPageViewModel extends ReactiveViewModel {
-  bool _lightTheme = _localStorageServices.darkMode;
+  bool _lightTheme = false;
```
Comment (isaacadariku) [inline lib/ui/views/business/settings/settings_page/settings_page_viewmodel.dart:11]: Why did you change this to false?

File: lib/core/data_sources/transaction/transaction_local_data_source.dart [hunk 2]
```diff
@@ -44,8 +44,8 @@ class TransactionLocalDataSourceImpl extends TransactionDataSource with Reactive
   List<String> formattedate = [];
   String date;
   
-  get box => _hiveService.openBox<TransactionModel>(HiveBox.transaction);
-  //var box = Hive.openBox<TransactionModel>(_boxname);
+  // get box => _hiveService.openBox<TransactionModel>(HiveBox.transaction);
```
Comment (isaacadariku) [inline lib/core/data_sources/transaction/transaction_local_data_source.dart:47]: Was this changes needed at your own part

File: lib/ui/views/business/settings/settings_page/settings_page_viewmodel.dart (hunk 1 above)
Comment (isaacadariku) [reply lib/ui/views/business/settings/settings_page/settings_page_viewmodel.dart:11]: @ufe-pr Please check this

Comment (Jonyango) [reply lib/ui/views/business/settings/settings_page/settings_page_viewmodel.dart]: I changed it as I was testing out whether the data was being persisted. When I was printing the trace, it was returning null.

File: lib/core/data_sources/transaction/transaction_local_data_source.dart (hunk 2 above)
Comment (Jonyango) [reply lib/core/data_sources/transaction/transaction_local_data_source.dart]: The function was raising an exception. I was figuring out why. The change wasn't necessary though as both methods kept raising exceptions.
